action.yml

```diff
@@ -16,10 +16,17 @@ inputs:
     default: |
       MATLAB
       Parallel_Computing_Toolbox
+  use-cache:
+    description: >-
+      Big blah
```

Comment:
Is this going to visible to users if we were to create a release?

Reply:
Oops, I forgot to fill that in with a real description. Fixed